modules/apps/portal-security/portal-security-ldap-impl/src/main/java/com/liferay/portal/security/ldap/internal/SafePortalLDAPImpl.java

```diff
@@ -495,6 +495,17 @@ public Binding getUser(
 			String emailAddress, boolean checkOriginalEmail)
 		throws Exception {
 
+		return getUser(
+			ldapServerId, companyId, screenName, emailAddress, false, true);
+	}
+
+	@Override
+	public Binding getUser(
+			long ldapServerId, long companyId, String screenName,
+			String emailAddress, boolean checkOriginalEmail,
+			boolean useUserSearchSafeLdapFilter)
+		throws Exception {
```

Comment:
Brian usually says that breaking changes are fine so instead of creating a new function change the existing one and add the new parameters there.